supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigBackendLifecycleActorFactory.scala

```diff
@@ -26,16 +25,12 @@ class ConfigBackendLifecycleActorFactory(val configurationDescriptor: BackendCon
       classOf[DispatchedConfigAsyncJobExecutionActor]
   }
 
-  // TODO: Wire in the funky configuration options too
-  override def runtimeAttributeDefinitions: Set[RuntimeAttributeDefinition] = {
-    import cromwell.backend.validation.RuntimeAttributesKeys._
-    Set(
-      RuntimeAttributeDefinition(DockerKey, required = false, None, usedInCallCaching = true),
-      RuntimeAttributeDefinition(ContinueOnReturnCodeKey, required = false, Some(WdlBoolean(false)), usedInCallCaching = true),
-      RuntimeAttributeDefinition(CpuKey, required = false, None, usedInCallCaching = false),
-      RuntimeAttributeDefinition(FailOnStderrKey, required = false, Some(WdlBoolean(true)), usedInCallCaching = true),
-      RuntimeAttributeDefinition(MemoryKey, required = false, None, usedInCallCaching = false)
-    )
+  override def runtimeAttributeDefinitions(initializationDataOption: Option[BackendInitializationData]):
+  Set[RuntimeAttributeDefinition] = {
+    val initializationData = BackendInitializationData.
+      as[SharedFileSystemBackendInitializationData](initializationDataOption)
```

Comment:
I'm open to a better underlying implementation of `as[_]`, as the current one was a quick and dirty way of unpacking the `Option` and casting back to the correct type. A richer `as[_]` could provide better error messages than the current `ClassCastException` but as I don't expect it to ever, ever happen, so I shrugged and moved on.